### PR TITLE
Add use hostname counter flag to NetworkEnvironment

### PR DESCRIPTION
**Pull request by szok**
- Add new flag use hostname counter to NetworkEnvironment
- Add next_hostname_without_model_counter method which return hostname based on already added hostname

### CODE REVIEW

File: src/ralph/networks/models/networks.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ class NetworkEnvironment(TimeStampMixin, NamedMixin, models.Model):
         blank=True,
         null=True,
     )
+    use_hostname_counter = models.BooleanField(default=True)
```
Comment (mkurek) [inline src/ralph/networks/models/networks.py]: add some help text here

File: src/ralph/networks/models/networks.py [hunk 2]
```diff
@@ -93,21 +94,65 @@ def next_free_hostname(self):
...
+        if hostnames:
+            hostnames = sorted(hostnames, reverse=True)
+            counter = int(hostnames[0]) + 1
+        hostname = AssetLastHostname(
+            prefix=self.hostname_template_prefix,
+            counter=counter,
+            postfix=self.hostname_template_postfix
+        )
+        if create_last_hostname:
+            hostname.save()
```
Comment (mkurek) [inline src/ralph/networks/models/networks.py]: we don't need to save (create?) `AssetLastHostname` when `use_hostname_counter` is `False`

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**